### PR TITLE
testing/rpm2cpio: new aport

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

**Comment by tcely** [comment]
A `check` function that downloads a RPM and checks that `cpio -it` can list the output of this script seems like a good addition to me.

**Comment by TBK** [comment]
@tcely sounds like a good idea. I try to find a suitable small rpm and make the check function.

**Comment by tcely** [comment]
I'd suggest something like http://mirror.centos.org/centos/7/os/x86_64/Packages/centos-bookmarks-7-1.el7.noarch.rpm

File: testing/rpm2cpio/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pkgrel=0
+pkgdesc="Convert .rpm files to cpio format"
+url="https://www.freshports.org/archivers/rpm2cpio/"
+arch="noarch"
+license="BSD-2-Clause"
+options="!check" # No test suite
+depends="libarchive-tools"
+source="rpm2cpio.sh"
+
+build() {
```
Comment (Ikke) [inline testing/rpm2cpio/APKBUILD]: This `build` function can be removed when it does nothing.

Comment (TBK) [reply testing/rpm2cpio/APKBUILD]: @Ikke Fixed.

**Pull request closed without merging**